packages/flutter/CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## [3.2.0](https://github.com/parse-community/Parse-SDK-Flutter/compare/V3.1.1...flutter-3.2.0) (2022-06-22)
```

Comment:
The latest flutter tag is `flutter-3.1.2`. Since this PR only contains bug fixes, the next flutter release will be `flutter-3.1.3`.

Reply:
I explained about this in the previous PR.

As we have dependencies update, some changing the MAJOR, I always change the MINOR, as it can break applications due to conflicting versions.

Version 3.2 is correct.

Reply:
A dependency major version change leads to a Parse Flutter SDK major version change in the following cases:
a) the dependency is transitive, or
b) the dependency is not transitive + the change requires the developer to change their custom code when interacting with the Parse Flutter SDK

If the dependency is transitive (which I assume from your comment?), and it has a major version increment, what is the reasoning to only increment the Parse SDK by a minor version? A breaking change in a transitive dependency breaks code and therefore requires a major version increment in the Parse SDK as well.

Reply:
This question needs to be resolved in order to determine the correct version. From what I see in this PR, this should bump the version to 4.0.0.

Reply:
Resolved in https://github.com/parse-community/Parse-SDK-Flutter/pull/762#issuecomment-1152275768